$NetBSD: patch-ab,v 1.3 2011/04/01 14:16:55 taca Exp $

* Correct URL which is strictly checked by RubyGems.
* Fix more gemspec format.

--- ../stream-0.5.gemspec.orig	2011-03-31 12:14:40.000000000 +0900
+++ ../stream-0.5.gemspec	2011-03-31 12:16:30.000000000 +0900
@@ -9,12 +9,16 @@
 cert_chain: 
 date: 2004-05-13 23:08:47.880344 +00:00
 default_executable: 
-dependencies: 
+dependencies: []
+
 description: Module Stream defines an interface for external iterators.
 email: hd.at.clr@hduchene.de
-executables: 
-extensions: 
-extra_rdoc_files: 
+executables: []
+
+extensions: []
+
+extra_rdoc_files: []
+
 files: 
 - install.rb
 - Rakefile
@@ -27,19 +31,21 @@
 - examples/examples.rb
 - examples/streamtester.rb
 has_rdoc: true
-homepage: rgl.rubyforge.org
+homepage: http://rgl.rubyforge.org
 licenses: 
 post_install_message: 
-rdoc_options: 
+rdoc_options: []
+
 require_paths: 
 - lib
 required_ruby_version: 
 required_rubygems_version: 
-requirements: 
+requirements: []
+
 rubyforge_project: rgl
 rubygems_version: 1.6.2
 signing_key: 
 specification_version: -1
 summary: Stream - Extended External Iterators
-test_files: 
+test_files: []
 
